symcache/src/breakpad.rs

```diff
@@ -37,8 +37,8 @@ impl<'input> BreakpadInfo<'input> {
 
     fn parse(&mut self, object: &'input Object) -> Result<()> {
         let mut records = object.breakpad_records();
-        while let Some(Ok(record)) = records.next() {
-            match record {
+        while let Some(record) = records.next() {
```

Comment:
Having `if let Some(Ok(foo)) = ...` should probably be a red flag in future code reviews.